openreview/conference/templates/SUPERpaperBidWebfield.js

```diff
@@ -0,0 +1,518 @@
+// webfield_template
+// Remove line above if you don't want this page to be overwriten
+
+// ------------------------------------
+// Add Bid Interface
+// ------------------------------------
+
+var CONFERENCE_ID = '';
+var HEADER = {};
+var SHORT_PHRASE = '';
+var BLIND_SUBMISSION_ID = '';
+var BID_ID = '';
+var SUBJECT_AREAS = '';
+var CONFLICT_SCORE_ID = '';
+var SCORE_IDS = [];
+var ENABLE_SUPER_ALGO = true;
+var BID_OPTIONS = [ 'Very Low', 'Low', 'Neutral', 'High', 'Very High' ];
+var POSITIVE_BIDS = [ 'High', 'Very High' ];
```

Comment:
This is just an example, I'm not planning to hard code these options. I saw this in your code:
```js
for (var key in bidsCount[noteId]) {
        if (key === 'High' || key === 'Very High') {  // <--- This line here
          var value = bidsCount[noteId][key];
          numberOfBids = numberOfBids+value;
        }
      }
```
Where High and Very High were hard coded, so I changed that to be POSITIVE_BIDS so that it could be parametrized too. That way if organizers want BID_OPTIONS to be:
`[ 'I like a lot', 'I like it', 'I like it a little bit', 'I don't like it', 'I hate it' ]`
Then the POSITIVE_BIDS can be:
`[ 'I like a lot', 'I like it', 'I like it a little bit' ]`
and the algorithm would work. Is this correct?

Reply:
Yes that is correct, looks good to me.